core/src/main/java/com/google/zxing/ResultMetadataType.java

```diff
@@ -93,5 +93,15 @@
    * parity is given with it.
    */
   STRUCTURED_APPEND_PARITY,
-  
+
+  /**
+   * For Code39 this contains the decoded contents with the Full-ASCII range.
+   */
+  CODE_39_EXTENDED,
```

Comment:
Ah yeah, just found that Code 93 also has this idea, so may not even be right to restrict this to Code 39 in the name: http://en.wikipedia.org/wiki/Code_93